### PR TITLE
routing/route+multi: remove redundant TotalFees field

### PR DESCRIPTION
**Pull request by halseth**
Instead get it on demand using method TotalFees().

As pointed out in https://github.com/lightningnetwork/lnd/pull/3064#discussion_r283911175

### CODE REVIEW

File: channeldb/payments.go [hunk 1]
```diff
@@ -361,7 +361,7 @@ func deserializeHop(r io.Reader) (*route.Hop, error) {
 
 func serializeRoute(w io.Writer, r route.Route) error {
 	if err := WriteElements(w,
-		r.TotalTimeLock, r.TotalFees, r.TotalAmount, r.SourcePubKey[:],
+		r.TotalTimeLock, r.TotalAmount, r.SourcePubKey[:],
```
Comment (halseth) [inline channeldb/payments.go]: note: this is a safe change since the code is not yet used anywhere.

Comment (wpaulino) [reply channeldb/payments.go]: Phew, was wondering why we weren't handling that 😅

**Comment by halseth** [comment]
Pushed a fixup commit to address an empty route struct. PTAL

**Comment by cfromknecht** [comment]
needs rebase tho

**Comment by halseth** [comment]
Rebased.

**Pull request merged**